roles/ansible-tower/tasks/main.yml

```diff
@@ -17,6 +17,7 @@
       - git
       - sshpass
       - ansible-2.6.6
+      - nano
```

Comment:
Any particular reason that nano is being installed?  The lab doesn't require the students to connect to the tower server via SSH.

Reply:
The main reason is to allow for impromtu additions to the workshop, or more frequently, a quick one-on-one during "free time".  I've used it a few times to show how to modify the "local playbooks and Tower"

Reply:
Sounds good to me.  Allowing them to login to the Tower command line might be relevant for some future advanced labs also.